Sources/Widgets/Representations/ResliceCursorContextRepresentation/index.js

```diff
@@ -136,13 +144,18 @@ function vtkResliceCursorContextRepresentation(publicAPI, model) {
     ctxVisible,
     hVisible
   ) => {
-    if (renderingType === RenderingTypes.PICKING_BUFFER) {
-      publicAPI.getActors().forEach((actor) => actor.setVisibility(wVisible));
-    } else {
-      publicAPI
-        .getActors()
-        .forEach((actor) => actor.setVisibility(wVisible && hVisible));
-    }
+    const visiblity =
+      renderingType === RenderingTypes.PICKING_BUFFER
+        ? wVisible
+        : wVisible && hVisible;
+
+    publicAPI.getActors().forEach((actor) => {
+      if (model.rotationActorsList.some((rotActor) => rotActor === actor)) {
+        actor.setVisibility(model.enableRotation);
```

Comment:
is visibility enough ? how about mouse interactions ?

Reply:
As I remember, the picker only pick visible actors. So, as soon as the actors are not visible, then no actions will be called.